src/main/java/duke/file/FileOperation.java

```diff
@@ -0,0 +1,120 @@
+package duke.file;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.StringTokenizer;
+
+public class FileOperation {
+    public static String relativeFileName = "duke.txt";
+    public static final String SEPARATOR = "|";
+
+    public static void readFile(ArrayList<Task> userTasks){
+        char taskSelection;
+        try {
+            File myObj = new File(relativeFileName);
```

Comment:
You should probably avoid naming the intermediate variable to something like "myObj". Try to give it a name that explains its usage.